bundlex.exs

```diff
@@ -16,7 +35,7 @@ defmodule Membrane.ExLibSRTP.BundlexProject do
           "srtp_util.c",
           "unifex_util.c"
         ],
-        pkg_configs: ["libsrtp2"],
+        os_deps: [{get_srtp_url(), "libsrtp2"}],
```

Comment:
```suggestion
        os_deps: [{[get_srtp_url(), :pkg_config], "libsrtp2"}],
```

Otherwise, it won't try finding the system library if there's no precompiled available